variables/advection/catalog.json

```diff
@@ -10,17 +10,35 @@
       "type": "application/json",
       "title": "Open Science Catalog"
     },
+    {
+      "rel": "parent",
+      "href": "../catalog.json",
+      "type": "application/json",
+      "title": "Variables"
+    },
     {
       "rel": "via",
-      "href": "https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/0cb7f2c6-5e99-4781-8d4f-19ecbad2e2e0?gtm_keyword=ADVECTION&gtm_scheme=Earth%20Science; https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/5f6358aa-872c-4c1c-9388-4714138f034a?gtm_keyword=ADVECTION&gtm_scheme=Earth%20Science; https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/30fd009d-df91-47ba-8800-2f2771f15e80?gtm_keyword=ADVECTION&gtm_scheme=Earth%20Science;https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/ce546f0d-d2e1-43ed-b8e0-a9079c690c56?gtm_keyword=ADVECTION&gtm_scheme=Earth%20Science",
+      "href": "https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/30fd009d-df91-47ba-8800-2f2771f15e80?gtm_keyword=ADVECTION&gtm_scheme=Earth%20Science",
       "type": "text/html",
       "title": "Description"
     },
     {
-      "rel": "parent",
-      "href": "../catalog.json",
-      "type": "application/json",
-      "title": "Variables"
+      "rel": "via",
+      "href": "https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/30fd009d-df91-47ba-8800-2f2771f15e80?gtm_keyword=ADVECTION&gtm_scheme=Earth%20Science",
+      "type": "text/html",
+      "title": "Description"
+    },
+    {
+      "rel": "via",
+      "href": "https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/30fd009d-df91-47ba-8800-2f2771f15e80?gtm_keyword=ADVECTION&gtm_scheme=Earth%20Science",
+      "type": "text/html",
+      "title": "Description"
+    },
```

Comment:
There seem to be duplicates in the Description URLs. We should only use unique URLs imho

Reply:
I see, I refactored the script and it should work now, I'll remove the current branch and opena new one